recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+cmake -G "Unix Makefiles"
```

Comment:
There might be several CMake knobs to tweak: https://github.com/MiKTeX/miktex/blob/b62a1e7ebe6b695b7a5ea7adbbe24635494a4d26/CMakeLists.txt